common/system/os.go

```diff
@@ -21,8 +22,9 @@ const (
 )
 
 var (
-	Arch = runtime.GOARCH
-	Os   = runtime.GOOS
+	Arch           = runtime.GOARCH
+	Os             = runtime.GOOS
+	SupportedArchs = []string{"x86_64", "aarch64", "arm", "i686"}
```

Comment:
I think that Apple Silicon has `arm64` answer as well for macOS

Reply:
https://github.com/lukso-network/tools-lukso-cli/blob/b51ae5a04438f52fa45005904abf78fe3514d89d/install/install.sh#L204 archs were based on the build that is supported for the CLI, not all the possible archs. See releases: https://github.com/lukso-network/tools-lukso-cli/releases/tag/v0.22.0